.github/workflows/main_win.yaml

```diff
@@ -58,6 +58,9 @@ jobs:
         )
       )
     steps:
+      - name: Set VC
+        run: set vc
```

Comment:
Lets add a comment and maybe link to origin as to what this means. E.g sets `Virtual Circuit` option for DNS lookup, to prevent failures XYZ